config/services.yml.erb

```diff
@@ -43,19 +43,25 @@ cwic_root: "http://cwic.wgiss.ceos.org/"
 edsc:
   development:
     enable_esi_order_chunking: true
+    reverb_root: "echo-reverb-rails.dev"
   test:
     enable_esi_order_chunking: true
+    reverb_root: "echo-reverb-rails.dev"
   sit:
     limited_collections: C1002-LPDAAC_TBD
     enable_esi_order_chunking: true
+    reverb_root: "testbed.echo.nasa.gov"
   uat:
     limited_collections: C1070-PVC_TS2, C1000-LPDAAC_TS2, C1052-LPDAAC_TS1, C1218664486-CUMULUS, C1216391161-LPCUMULUS, C1071-PVC_TS2, C98791-LPDAAC_TS1, C1216100909-LPDAAC_TS2, C1216257556-LPDAAC_TS1, C1216101022-LPDAAC_TS2, C1216257557-LPDAAC_TS1, C1216101026-LPDAAC_TS2, C1216257558-LPDAAC_TS1, C1216101028-LPDAAC_TS2, C1216257559-LPDAAC_TS1, C1216101030-LPDAAC_TS2, C1216257560-LPDAAC_TS1, C1216101032-LPDAAC_TS2, C1216257561-LPDAAC_TS1, C1216101034-LPDAAC_TS2, C1216257562-LPDAAC_TS1, C1216101036-LPDAAC_TS2, C1216257563-LPDAAC_TS1, C1216101038-LPDAAC_TS2, C1216257593-LPDAAC_TS1, C1216101040-LPDAAC_TS2, C1216257565-LPDAAC_TS1
     enable_esi_order_chunking: true
+    reverb_root: "api-test.echo.nasa.gov"
   lab:
     enable_esi_order_chunking: true
+    reverb_root: "testbed.echo.nasa.gov"
   production:
     limited_collections: C14758250-LPDAAC_ECS, C190733714-LPDAAC_ECS, C1299783607-LPDAAC_ECS, C1299783800-LPDAAC_ECS, C1299783608-LPDAAC_ECS, C1299783630-LPDAAC_ECS, C1299783574-LPDAAC_ECS, C1299783609-LPDAAC_ECS, C1299783631-LPDAAC_ECS, C1299783579-LPDAAC_ECS, C1299783651-LPDAAC_ECS, C1299783684-LPDAAC_ECS
     enable_esi_order_chunking: true
+    reverb_root: "reverb.echo.nasa.gov"
```

Comment:
*I think* these should be full URLs starting with ```https?:\\```

Reply:
I had stripped them as the referrer is regex'd so as to be missing the leading https:// - I updated the code so that the ENV values are also regex'd before comparison and added the leading http?:\\ back - 'should' work.

Reply:
It will work for sure. I can't remember 100% but I think if you put the link in href, it may treat the link as a relative url without the leading http.